scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt

```diff
@@ -223,6 +225,20 @@ class ClearlyDefinedStorageTest : WordSpec({
     }
 
     "ClearlyDefinedStorage" should {
+        "handle a SocketTimeoutException" {
+            server.stubFor(
+                get(anyUrl())
+                    .willReturn(aResponse().withFixedDelay(2000))
+            )
+            val client = OkHttpClientHelper.buildClient {
+                readTimeout(Duration.ofSeconds(1))
```

Comment:
Is execution time determined by this 1 second? If so, would it make sense to further reduce it, like a couple of millis or so?

Reply:
TBH, I haven't bothered to think about this as I took the code straight from

https://github.com/oss-review-toolkit/ort/blob/a37f8baff437059e3b2f940dcce88762158bb841/analyzer/src/test/kotlin/curation/ClearlyDefinedPackageCurationProviderTest.kt#L61-L77

Maybe @oheger-bosch, who authored this, has an opinion?

Reply:
I'm fine if you just merge it. If exec time can be significantly reduced, that'd be a separate commit anyway as there is another place where this would be fixed.

Reply:
Yes, the timeout directly impacts the execution time.

Reply:
> Yes, the timeout directly impacts the execution time.

And do you think it's safe to further decrease it to reduce test execution time?

Reply:
> > Yes, the timeout directly impacts the execution time.
> 
> And do you think it's safe to further decrease it to reduce test execution time?

I assume, yes. Probably the HTTP client starts a timer to interrupt the request when the timeout occurs.

Reply:
Ok, thanks, see https://github.com/oss-review-toolkit/ort/pull/6582.